install/kubernetes/helm/subcharts/tracing/templates/jaeger/io_v1alpha1_jaeger_crd.yaml

```diff
@@ -0,0 +1,15 @@
+{{ if eq .Values.provider "jaeger" }}
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
@sdake i know you've been looking at CRD install. Does this approach pass muster, given what you've been looking at? Are there ordering concerns?

Reply:
CRDs should go in the file install/helm/kubernetes/istio-init/files/ and a job should go in the templates dir to apply the crd.  Take a look at the crd-init chart - should be straightforward enough to understand.